index.js

```diff
@@ -81,7 +81,7 @@ const handleArguments = (rawFile, rawArgs, rawOptions = {}) => {
 };
 
 const handleOutput = (options, value) => {
-	if (typeof value !== 'string' && !ArrayBuffer.isView(value)) {
+	if (value === undefined || value === null) {
```

Comment:
`value` is `null` only when using `execaSync()`, as this is what `child_process.spawnSync()` returns for `result.stdout|stderr` and `result.output[*]`.